t/12_sles4sap_publiccloud.t

```diff
@@ -136,6 +136,94 @@ subtest "[sles4sap_cleanup] no need to clean" => sub {
     ok($ret eq 0, "Expected return 0 ret:$ret");
 };
 
+subtest "[sles4sap_cleanup] SUPPORTCONFIG=0 => never collect, even on FAIL" => sub {
+    my $sles4sap_publiccloud = Test::MockModule->new('sles4sap_publiccloud', no_auto => 1);
+    $sles4sap_publiccloud->redefine(select_host_console => sub { return; });
```

Comment:
@lpalovsky has an even nicer pattern for it using `noop` https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/23965/files#diff-8ab970ed838795161ca18f2044e73c143d072b3f379b3a247dd8582fd307453bR83 

(not for this PR)